drv/stm32xx-i2c/src/lib.rs

```diff
@@ -203,8 +203,16 @@ enum Trace {
     Stop,
     RepeatedStart(#[count(children)] bool),
     LostInterrupt,
+    #[count(skip)]
     Panic(Register, u32),
     #[count(skip)]
```

Comment:
I added `#[count(skip)]` to the ringbuf entries that are only recorded on panics, since we don't really care about counting them --- they'll either be counted exactly the number of times we record them in the panic code, or they'll never be recorded at all. So, I figured it was worth saving the 8 bytes we can shave off by not generating counters for these.